google/cloud/optional.h

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_OPTIONAL_H_
-#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_OPTIONAL_H_
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_OPTIONAL_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_OPTIONAL_H_
 
-#include "google/cloud/internal/throw_delegate.h"
+#include "internal/throw_delegate.h"
```

Comment:
To be consistent with other files, I think this include path should start relative to the root (and thus have the `google/cloud/` prefix), right?

Reply:
Yes. Fixed.